labels.json

```diff
@@ -4436,7 +4556,12 @@
                 "lang": "en",
                 "name": "Cockatiel",
                 "description": "This user is a Cockatiel! [Category: Avian/Bird]"
-            }
+            },
+            {
+            	"lang": "pt-BR",
+            	"name": "Calopsita",
+            	"description": "Este usuário é uma Calopsita! AKA: caturra"
```

Comment:
+ `[Category: Avian/Bird]`

Reply:
My bad! I think it cut it instead of copying.